src/improvements/tasks/StateOverrideManager.sol

```diff
@@ -77,18 +78,20 @@ abstract contract StateOverrideManager is CommonBase {
             for (uint256 j = 0; j < _stateOverrides[i].overrides.length; j++) {
                 if (_stateOverrides[i].overrides[j].key == nonceSlot) {
                     uint256 userDefinedNonce = uint256(_stateOverrides[i].overrides[j].value);
-                    // This is an important safety check. Users should not be able to set the nonce to a value less than the current actual nonce.
-                    require(
-                        userDefinedNonce >= currentActualNonce,
-                        string.concat(
-                            "StateOverrideManager: User-defined nonce (",
-                            vm.toString(userDefinedNonce),
-                            ") is less than current actual nonce (",
-                            vm.toString(currentActualNonce),
-                            ") for contract: ",
-                            vm.toString(safeAddress)
-                        )
-                    );
+                    if (Utils.isFeatureEnabled("DISABLE_OVERRIDE_NONCE_CHECK")) {
```

Comment:
Can you put a comment in here describing where this env var would be set and why?

Reply:
Done in commit: [4e9a407](https://github.com/ethereum-optimism/superchain-ops/pull/1051/commits/4e9a4074e888e8aa33c9781bba85e7a77fd32ad0)